File: my-folder/0027-remove-element/solution.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int k =0;
        for(size_t i = 0;i<nums.size();++i){
            if(nums[i] == val){
                nums[i] = 1e8+2;
                continue;
            }
            k++;
        }
        sort(nums.begin(),nums.end());
        return k;
    }
};

File: my-folder/0027-remove-element/solution.ts
function removeElement(nums: number[], val: number): number {
    for(let i = 0; i < nums.length; ++i){
        let lastIndex = nums.length - 1;
        if(nums[i] === val){
            nums[i] = nums[lastIndex]; 
            nums.pop();
            i--;
        }
    }

    return nums.length;
};
